src/Workspace/index.jsx

```diff
@@ -370,6 +371,13 @@ class Workspace extends React.Component {
         if (this.workspaceStates.includes(data.status)) {
           const workspaceLaunchStepsConfig = this.getWorkspaceLaunchSteps(data);
           let workspaceStatus = data.status;
+          if (args?.triggerPayModelCall && workspaceStatus === 'Not Found') {
```

Comment:
minor: might be better to read and reduce complexityu a bit by just passing in a boolean arg called `triggerPayModelCall` instead of expecting a generic `arg` object and parse it, unless there is a specific reason to do so?

Reply:
I thought this could be better to extend the functionality in the future with optional params.